docs/usecases/testing.md

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
This page also needs to be added in https://github.com/zio/zio/blob/master/website/sidebars.json